File: 0128-longest-consecutive-sequence/0128-longest-consecutive-sequence.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
       if(nums.size()==0) return 0; 
       sort(nums.begin(),nums.end());
       int count =1;
       int maxi =1;
       for(int i=1;i<nums.size();i++){
        if(nums[i]==nums[i-1]+1){
            count+=1;
            maxi=max(maxi,count);
        }else if(nums[i]!=nums[i-1]){
            count=1;
        }
       }
       return maxi;
    }
};

File: 0128-longest-consecutive-sequence/0128-longest-consecutive-sequence.py
from collections import Counter
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums.sort()
        maxi = 1
        cnt = 1
        if not nums:
            return 0
        for i in range(1,len(nums)):
            if(nums[i] ==nums[i-1]+1):
                cnt+=1  
            elif nums[i] == nums[i - 1] :
                continue 
            else:
                cnt=1
            maxi = max(maxi,cnt)
        return maxi